test/Elastic.Apm.AspNetCore.Tests/MicrosoftExtensionsConfigTests.cs

```diff
@@ -109,7 +113,7 @@ public void ReadConfingsFromEnvVarsViaIConfig()
 				.AddEnvironmentVariables()
 				.Build();
 
-			var config = new MicrosoftExtensionsConfig(configBuilder, new TestLogger());
+			var config = new MicrosoftExtensionsConfig(configBuilder, new NoopLogger(), "test");
```

Comment:
Please see my comment above regarding adding an `internal` constructor overload to `MicrosoftExtensionsConfig`.